nearcore/src/runtime/mod.rs

```diff
@@ -562,9 +563,18 @@ impl NightshadeRuntime {
                 RuntimeError::ReceiptValidationError(e) => panic!("{}", e),
                 RuntimeError::ValidatorError(e) => e.into(),
             })?;
+        let elapsed = instant.elapsed();
 
         let total_gas_burnt =
             apply_result.outcomes.iter().map(|tx_result| tx_result.outcome.gas_burnt).sum();
+        metrics::APPLY_CHUNK_DELAY
+            .with_label_values(&[&format_total_gas_burnt(total_gas_burnt)])
```

Comment:
Won't this result in an explosion in cardinality, since the label value is going to be pretty much always different every time a chunk is processed? I believe it would effectively cause the prometheus instance to stop behaving quite quickly, and neard would also appear to leak memory since it'd need to store additional data in memory for each chunk processed.

Reply:
13 labels is a very reasonable amount, I didn't notice the rounding, mostly because I missed the function being added.